corehq/apps/fixtures/templates/fixtures/view_table.html

```diff
@@ -100,44 +104,29 @@ <h2 class="panel-title">
 {% endblock %}
 {% block posttable %}{% endblock %}
 {% endblock %}
-{% endblock %}
-
-{% block js-inline %} {{ block.super }}
-    <script type="text/javascript">
-
+<div id="initial-page-data" class="hide">
+    {
         {% if report_table and report_table.datatables %}
-            var reportTables = new HQReportDataTables({
-                dataTableElem: '#report_table_{{ report.slug }}',
-                defaultRows: {{ report_table.default_rows|default:10 }},
-                startAtRowNum: {{ report_table.start_at_row|default:0 }},
-                showAllRowsOption: {{ report_table.show_all_rows|JSON }},
-
-                {% if report_table.headers.render_aoColumns %}aoColumns: {{ report_table.headers.render_aoColumns|JSON }},{% endif %}
-                autoWidth: {{ report_table.headers.auto_width|JSON }},
-                {% if report_table.headers.custom_sort %}customSort: {{ report_table.headers.custom_sort|JSON }},{% endif %}
-
+            "renderReportTables": 1,
+            "reportTablesOptions": {
+                "slug": "{{ report.slug }}",
+                "defaultRows": {{ report_table.default_rows|default:10 }},
+                "startAtRowNum": {{ report_table.start_at_row|default:0 }},
+                "showAllRowsOption": {{ report_table.show_all_rows|JSON }},
+                "autoWidth": {{ report_table.headers.auto_width|JSON }}
+                {% if report_table.headers.render_aoColumns %}, "aoColumns": {{ report_table.headers.render_aoColumns|JSON }}{% endif %}
+                {% if report_table.headers.custom_sort %}, "customSort": {{ report_table.headers.custom_sort|JSON }}{% endif %}
                 {% if report_table.pagination.is_on %}
-                    ajaxSource: '{{ report_table.pagination.source }}',
-                    ajaxParams: {{ report_table.pagination.params|JSON }},
+                    , "ajaxSource": "{{ report_table.pagination.source }}"
```

Comment:
Comma-first notation helpful to avoid a trailing comma, which will choke the JSON parser.